include/aspect/simulator_access.h

```diff
@@ -64,6 +64,10 @@ namespace aspect
   {
     template <int dim> class Interface;
   }
+  namespace AdiabaticConditions
+  {
+    template <int dim> class Interface;
+  }
```

Comment:
How did we not need this before? You're not making any other changes to this file...